### PR TITLE
testing changes and adding a todo for 2020 measles updates

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

**Review by alibow — APPROVED** [review]

File: docs/source/gbd2020_models/causes/measles/index.rst [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+   though  measles can lead to diarrhea or other causes that we include in our
+   Vivarium models, we won't be double counting mortality and morbidity from
+   these causes. For example, a death caused by diarrheal dehydration due to
+   measles should be counted in the GBD as a death due to measles, not as a
+   death due to diarrheal diseases.
+
+   The relationship with vitamin A deficiency may also be important for our
+   models.
+
+   Make sure to check on measles sequelae as well. Our models so far have not paid much attention to the nonfatal side, but it looks like some of the complications can persist well after someone recovers from measles, so maybe that's important to think about.
```
Comment (aflaxman) [inline docs/source/gbd2020_models/causes/measles/index.rst:85]: There is some really interesting stuff that goes beyond GBD causal framework about how measles can "reset" the immune system and put people at higher risk of a range of diseases later --- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8590458/

File: docs/source/gbd2020_models/causes/measles/index.rst [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+   * - prevalence_s{`sid`}
+     - como
+     - step5
+     - Prevalence of each sequelae
+     - 
+
+Validation Criteria
+-------------------
+
+Simulation results should replicate the GBD 2019 cause-specific mortality rate, incidence rate, and prevalence for all age/sex/location groups. Notably, these measures should be tracked over time in the simulation to ensure that simulation rates do not deviate from GBD rates as the simulation progresses.
```
Comment (aflaxman) [inline docs/source/gbd2020_models/causes/measles/index.rst:374]:
```suggestion
Simulation results should replicate the GBD 2020 cause-specific mortality rate, incidence rate, and prevalence for all age/sex/location groups. Notably, these measures should be tracked over time in the simulation to ensure that simulation rates do not deviate from GBD rates as the simulation progresses.
```

**Review by plinds — APPROVED** [review]

File: docs/source/gbd2020_models/causes/measles/index.rst [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+   Retrieved 13 Nov 2019.
+   https://www.cdc.gov/vaccines/pubs/pinkbook/meas.html
+
+.. [Wikipedia-Measles-2020] Measles. From Wikipedia, the Free Encyclopedia.
+   Retrieved 13 Nov 2019.
+   https://en.wikipedia.org/wiki/Measles
+
+.. todo::
+
+   update this cite to 2019
```
Comment (aflaxman) [inline docs/source/gbd2020_models/causes/measles/index.rst:396]:
```suggestion
   update this cite to 2020
```

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2020_models/causes/measles/index.rst (hunk 1 above)
Comment (SylLutze) [reply docs/source/gbd2020_models/causes/measles/index.rst:85]: Thanks! I'll bookmark this for reference and add it into a later commit

**Pull request merged**